samples/net/sockets/civetweb/src/main.c

```diff
@@ -139,12 +146,12 @@ void *main_pthread(void *arg)
 {
 	static const char * const options[] = {
 		"listening_ports",
-		"8080",
+		XSTR(HTTP_PORT),
 		"num_threads",
 		"1",
 		"max_request_size",
-		"2048",
-		0
+		XSTR(MAX_REQUEST_SIZE_BYTES),
+		NULL
```

Comment:
Just use ```STRINGIFY``` here.

```c
static const char * const options[] = {
		"listening_ports",
		STRINGIFY(HTTP_PORT),
		"num_threads",
		"1",
		"max_request_size",
		STRINGIFY(MAX_REQUEST_SIZE_BYTES),
		NULL
	};
```

Seems to work and doesn't generate errors on my side ;)

Reply:
Same for me. Thank you very much for this advice!